index.md

```diff
@@ -1,4 +1,4 @@
 ---
 layout: index
-title: Quarkus - Supersonic Subatomic Java
+title: Supersonic Subatomic Java
```

Comment:
The "- Quarkus" is generated for all the pages. So the new title is "Supersonic Subatomic Java - Quarkus". If we don't change the index.md file, it will read "Quarkus - Supersonic Subatomic Java - Quarkus"

Reply:
No, we could tweak the template to not add it at the end if the title starts with Quarkus. Similar to what was done before but with 2 conditions.

I think it would be better that the home page actually says `Quarkus - Supersonic Subatomic Java`. I can do the legwork if you want.

Reply:
I'm good with that option. I've you could code it, that would be great.